community/boilerplates/event-triggers/aws-lambda/ruby/mutation/lambda_function.rb

```diff
@@ -0,0 +1,44 @@
+require 'json'
+require 'net/http'
+
+def lambda_handler(event:, context:)
+  access_key = ENV['ACCESS_KEY']
+  hge_endpoint = ENV['HGE_ENDPOINT']
+  query = <<~QUERY
+    mutation updateNoteRevision ($noteId: Int!, $data: String!) {
+      insert_note_revisions (objects: [
```

Comment:
This should be `insert_note_revision`. It worked for me after changing it to that 🙂